default/scripting/species/SP_ANCIENT_GUARDIANS.focs.txt

```diff
@@ -0,0 +1,81 @@
+Species
+    name = "SP_ANCIENT_GUARDIANS"
+    description = "SP_ANCIENT_GUARDIANS_DESC"
+    gameplay_description = "SP_ANCIENT_GUARDIANS_GAMEPLAY_DESC"
+
+    tags = [ "ROBOTIC" "AI_TAG_NO_INDUSTRY" "AI_TAG_NO_RESEARCH" "AI_TAG_AVERAGE_SUPPLY"]
```

Comment:
The AI_TAG_ prefix was removed in e39340464446e7bd3b268088ac4f0d03393a20e1. Just use "NO_INDUSTRY" etc.